### PR TITLE
callum june optimizations

### PR DESCRIPTION
**Pull request by ksatx**

### CODE REVIEW

**Comment by lottamus** [comment]
Deploy preview for *stoplightio* ready!

Built with commit 10a4cc40c3fdb12af5a7cffcf604f9e0bf2bec2a

https://deploy-preview-1477--stoplightio.netlify.app

File: netlify/landings/design.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ hero:
     src: /images/studio/hero.png
   subtitle: >-
     Build first-class APIs with a collaborative, powerful, visual editor that
-    allows you to prototype and share your API within minutes
+    allows you to prototype and share your API within minutes using Stoplights API design Tools.
```
Comment (paulatulis) [inline netlify/landings/design.yaml:12]:
```suggestion
    allows you to prototype and share your API within minutes using Stoplight's API design tools.
```

File: netlify/subpages/api-documentation-guide.md [hunk 2]
```diff
@@ -41,9 +41,9 @@ In this API documentation guide & tutorial, we’ll cover the basics of document
 
 ### What is API documentation?
 
-API docs are the collection of references, tutorials, and examples that help developers use your API.
+API docs, or API description documents, are the collection of references, tutorials, and examples that help developers use your API.
 
-Your API's document is the primary resource for explaining what is possible with your API and how to get started. It also serves as a place for developers to return with questions about syntax or functionality. Your API docs have these answers.
+Your API's document is the primary resource for explaining what is possible with your API and how to get started. It also serves as a place for developers to return with questions about syntax or functionality. The best API docs have these answers hence why it is so important to document your API.
```
Comment (paulatulis) [inline netlify/subpages/api-documentation-guide.md:46]:
```suggestion
Your API's documentation is the primary resource for explaining what is possible with your API and how to get started. It also serves as a place for developers to return with questions about syntax or functionality. The best API docs have these answers hence why it is so important to document your API.
```

File: netlify/subpages/mock-api-server-guide-advanced.md [hunk 3]
```diff
@@ -34,9 +34,9 @@ meta:
...
+The major requirement of a mock API REST server is to generate the data to send as responses. Ideally, the data is realistic, because you want to make the experience as close to a real API as you can. However, an overly complex effort to build a mock API may negate its purpose. In the cases where the real API is available, the mock API is meant to unblock projects that depend on the API, not become a project itself. We will cover all these points in this dummy REST API mock data mock rest API definition guide.
 
-When generating mock data, you can use static responses or fully randomized fields. Depending on how you create advanced REST API mock data, you may also be able to find a middle ground of mostly static content with some fields dynamically generated.
+When generating mock data, you can use static responses or fully randomized fields. Depending on how you create advanced REST API mock data and mocking rest services data, you may also be able to find a middle ground of mostly static content with some fields dynamically generated.
```
Comment (paulatulis) [inline netlify/subpages/mock-api-server-guide-advanced.md:39]:
```suggestion
When generating mock data, you can use static responses or fully randomized fields. Depending on how you create advanced REST API mock data and mocking REST services data, you may also be able to find a middle ground of mostly static content with some fields dynamically generated.
```

**Pull request merged**